kafka/src/test/groovy/io/micronaut/configuration/kafka/docs/consumer/batch/BookListener.java

```diff
@@ -28,7 +29,7 @@ public void receiveList(List<Book> books) { // <2>
 
     // tag::reactive[]
     @Topic("all-the-books")
-    public Flux<Book> receiveFlux(Flux<Book> books) {
+    public Publisher<Book> receiveFlux(Flux<Book> books) {
```

Comment:
The examples in this style with mixed usage of `Flux/Mono` and `Publisher` don't quite sit right with me. If a class is already importing Flux or Mono so as to use their more helpful features, what is the benefit of dropping down and using `Publisher` for the return type? I think I'd prefer to see this continue to return `Flux`.

Reply:
OK. Thanks you. I don't disagree, but will see what @Sergio wants to do. He would like to minimize use of library-specific PAIs but I don't always see how to get around it for examples.

Reply:
I think you tagged the wrong Sergio. :)